sdk/storage/storage-blob/README.md

```diff
@@ -235,27 +226,58 @@ async function streamToString(readableStream) {
     readableStream.on("error", reject);
   });
 }
+```
 
-// An async method returns a Promise object, which is compatible with then().catch() coding style.
-main()
-  .then(() => {
-    console.log("Successfully executed sample.");
-  })
-  .catch((err) => {
-    console.log(err.message);
-  });
+A complete example of basic scenarios is at [samples/basic.js](https://github.com/azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob/samples/basic.js).
+
+## Troubleshooting
+
+It could help diagnozing issue by turning on the console logging. Here's an example logger implementation.
+
+```javascript
+class ConsoleHttpPipelineLogger {
+  constructor(minimumLogLevel) {
+    this.minimumLogLevel = minimumLogLevel;
+  }
+  log(logLevel, message) {
+    const logMessage = `${new Date().toISOString()} ${HttpPipelineLogLevel[logLevel]}: ${message}`;
+    switch (logLevel) {
+      case HttpPipelineLogLevel.ERROR:
+        // tslint:disable-next-line:no-console
```

Comment:
This code is javascript, so we can remove tslint comments

Reply:
Good catch! Fixed.